src/main/content/_assets/css/doc-header.css

```diff
@@ -194,7 +193,7 @@
     }
 
     #navbar_responsive_button {
-        background-color: #313653;
+        background-image:linear-gradient(-1deg, #141427 0%, #2c2d4f 100%);        
```

Comment:
Is there a reason to use `background-image` over `background`?
It seems like both should work the same in this case.